### PR TITLE
feat: structure proposal

### PR DESCRIPTION
**Pull request by mayoforkovic**

### CODE REVIEW

**Review by FrantisekMichalSebestyen — CHANGES_REQUESTED** [review]

File: src/components/interactions/InteractionStory.vue [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <template>
     <div class="scroll-my-20" :id="active ? 'active-story' : undefined">
-        <img v-if="first" class="mx-auto mb-8 h-40 markdown" src="../img/interaction-intro-ester.svg" alt="Ester" />
+        <img v-if="first" class="mx-auto mb-8 h-40 markdown" src="../../img/interaction-intro-ester.svg" alt="Ester" />
         <img
             v-else-if="activeOrTransitioning"
             class="h-12 w-12 rounded-xl object-cover"
-            src="../img/avatar-ester.png"
+            src="../../img/avatar-ester.png"
```
[the remark this answers is not in the document]
Comment (mayoforkovic) [reply src/components/interactions/InteractionStory.vue:7]: it was fixed in last PR

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,7 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+auto-imports.d.ts
+components.d.ts
+/public/fonts/
```
[the remark this answers is not in the document]
Comment (igor-kamil) [reply .gitignore:30]: I don't really mind. But I think it was not necessary to change this 😉

File: src/components/interactions/InteractionStory.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (igor-kamil) [reply src/components/interactions/InteractionStory.vue:7]: confirmed 
https://github.com/SlovakNationalGallery/atlas-frontend/pull/5/commits/19e7083bad2aca5b67cc74b5f56da65248a975e0

**Review by igor-kamil — APPROVED** [review]
Overall well done. I like the structure more. Also router-link looks nicer after this change. Works nicely... for me  good to go 🚀 
For the next time: use the task ID in the branch name (for syncing with jira).
Also it's always nice to write something in the PR overview 👀

**Review by FrantisekMichalSebestyen — APPROVED** [review]

**Pull request merged**